### PR TITLE
Updated Readme 

### PR DESCRIPTION
**Pull request by richarddushime**
#1 updated readme for  setting up database instructions

### CODE REVIEW

**Comment by rowlandm** [comment]
Hi Richard,

This is already explained here that is much easier to do.

https://github.com/WEHI-ResearchComputing/student_intern_data_public

Check the Readme.

Why don't you give it a go :)

File: README.md [hunk 1]
```diff
@@ -1,8 +1,21 @@
...
 ## installation
+After Successfully setting up the student-intern-organiser Application repository 
+Do the Following to setup the Database
+1. Change Directory/Folder
+```
+cd student-intern-organiser
+```
+2. clone the repository
+```
+git clone https://github.com/WEHI-ResearchComputing/student_intern_data_public.git
```
Comment (rowlandm) [inline README.md]: You can do this all in one command:

git clone https://github.com/WEHI-ResearchComputing/student_intern_data_public.git student_intern_data

File: README.md [hunk 2]
```diff
@@ -1,8 +1,21 @@
...
+```
+mkdir student_intern_data/
+```
+4. Copy All from **student_intern_data_public** to **student_intern_data**
+
+Enjoy the Run
 
-- git clone git@github.com:WEHI-ResearchComputing/student-intern-organiser.git public_student_intern_organiser
-- cd public_student_intern_organiser/
-- git clone git@github.com:WEHI-ResearchComputing/student_intern_data_public.git student_intern_data
```
Comment (rowlandm) [inline README.md:7]: See this line - this does all that for you :)

**Review by rowlandm — CHANGES_REQUESTED** [review]
Just need to improve it to use the one line to create the directory

File: README.md (hunk 2 above)
Comment (richarddushime) [reply README.md:7]: yes  thats true , i was only changing from using ssh url to https url

**Review by rowlandm — APPROVED** [review]

**Comment by rowlandm** [comment]
Cheers Richard!

**Comment by richarddushime** [comment]
Thanks 💯

**Pull request merged**